### PR TITLE
Add 'listed' filed to output metadata.

### PR DESCRIPTION
**Pull request by kmaglione**

### CODE REVIEW

**Comment by magopian** [comment]
Maybe a quick test?

r+wc

File: validator/errorbundler.py [hunk 1]
```diff
@@ -60,7 +60,8 @@ def __init__(self, determined=True, listed=True, instant=False,
         self.pushable_resources = {}
         self.final_context = None
 
-        self.metadata = {'requires_chrome': False}
+        self.metadata = {'requires_chrome': False,
+                         'listed': listed}
```
Comment (magopian) [inline validator/errorbundler.py]: This fits on a single line

**Comment by kmaglione** [comment]
Added a couple of tests.

**Comment by magopian** [comment]
Excellent, thanks for the added tests! And it seems you have the commit bit now. Are you going to cut a new release or should I do it?

**Comment by kmaglione** [comment]
You should probably do it. I've never done a validator release before.

**Pull request merged**